File: metric_tool/test/resources/kotlin/lcam/LcamE.kt
package io.company.app.view.basket.kotlin.lcam

internal class LcamE(private val myField: String) {

    fun getField(): String {
        return this.myField
    }

    fun getFieldAsLocalVariable(): String {
        val myField = "field as local variable"
        return myField
    }

    fun getFieldWithParameterVariable(myField: String): String {
        return myField + myField
    }
}

File: metric_tool/test/resources/dart/lcam/lcam_e.dart
library mylib;

class LcamE {
  LcamE(String myField) : _myField = myField;

  final String _myField;

  String getField() {
    return _myField;
  }

  String getFieldAsLocalVariable() {
    String _myField = "field as local variable";
    return _myField;
  }
}
